src/view/treeNodes/pullRequestNode.ts

```diff
@@ -376,13 +377,18 @@ export class PRNode extends TreeNode {
 		const tooltip = `${tooltipPrefix}${title} (#${formattedPRNumber}) by @${login}`;
 		const description = `#${formattedPRNumber} by @${login}`;
 
+		const iconPath = {
+			light: mergeable ? 'resources/icons/light/check.svg' : 'resources/icons/delete.svg',
```

Comment:
for these paths, I think we need to use a VSCode API to find the absolute path for them, since VSCode will know where the extension is installed

`src/common/resources.ts` has code for this, you can add the check and delete icons there and then import them here

Reply:
Thanks for the hints, I was going to have to ask that question eventually.